stellar/contract/round/src/data_type.rs

```diff
@@ -0,0 +1,103 @@
+use crate::soroban_sdk::{self, contracttype, Address, String, Vec};
+
+#[contracttype]
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub enum ApplicationStatus {
+    Pending,
+    Approved,
+    Rejected,
+}
+
+#[contracttype]
+#[derive(Clone, Eq, PartialEq, Debug)]
+pub struct RoundDetail {
```

Comment:
keep the name of the functions and fields the same with the near one:

https://github.com/PotLock/grantpicks/blob/chore/initial-near-spec/near/contract/src/lib.rs#L59

Reply:
since the module name is round and contract struct is Round. it should be RoundDetail